### PR TITLE
Upgrade kafka-lib to the latest version, solve compatibility issues

### PR DESCRIPTION
**Pull request by Coopermasaaki**

### CODE REVIEW

File: server/entrance/server.go [hunk 1]
```diff
@@ -28,14 +28,26 @@ func (s *server) run(ctx context.Context, cfg *Config) error {
...
-		topics.RepoBranchFetched: s.handleRepoBranchFetched,
-		topics.RepoFileFetched:   s.handleRepoFileFetched,
+	t := []string{
+		topics.RepoFetched,
+		topics.RepoBranchFetched,
+		topics.RepoFileFetched,
 	}
 
-	return kafka.Subscribe(cfg.GroupName, h)
+	return kafka.Subscribe(cfg.GroupName, s.handleByHeader, t)
```
Comment (yangwei999) [inline server/entrance/server.go:37]: return kafka.Subscribe(cfg.GroupName, s.handleByHeader, []string{
    cfg.Topic.RepoFetched,
    ....,
})

File: server/infrastructure/messageimpl/impl.go [hunk 2]
```diff
@@ -17,18 +17,26 @@ type repoFileMessage struct {
 }
 
 func (p *repoFileMessage) SendRepoBranchFetchedEvent(e message.Message) error {
-	return send(p.topics.RepoBranchFetched, e)
+	header := map[string]string{
+		"header_key": "handleRepoBranchFetched",
```
Comment (yangwei999) [inline server/infrastructure/messageimpl/impl.go:21]: key也用常量吧

File: server/entrance/server.go (hunk 1 above)
Comment (Coopermasaaki) [reply server/entrance/server.go]: solved

File: server/infrastructure/messageimpl/impl.go (hunk 2 above)
Comment (Coopermasaaki) [reply server/infrastructure/messageimpl/impl.go]: solved

**Comment by yangwei999** [comment]
lgtm

File: server/entrance/server.go [hunk 3]
```diff
@@ -27,15 +26,28 @@ func (s *server) run(ctx context.Context, cfg *Config) error {
 }
 
 func (s *server) subscribe(cfg *Config) error {
-	topics := &cfg.Topics
+	err1 := kafka.Subscribe(cfg.GroupName, s.handleRepoFetched, []string{
+		cfg.Topics.RepoFetched,
+	})
+	if err1 != nil {
+		return err1
```
Comment (pkking) [inline server/entrance/server.go:33]: 可以复用 一个err

Comment (Coopermasaaki) [reply server/entrance/server.go]: done

**Pull request merged**